tests/unit_tests/test_wave_06.py

```diff
@@ -75,8 +75,17 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(tai.inventory) == len(jesse.inventory) == 3
+    assert len(tai.get_by_category("Clothing")) == 1
+    assert len(jesse.get_by_category("Clothing")) == 1
+    assert len(tai.get_by_category("Decor")) == 1
+    assert len(jesse.get_by_category("Decor")) == 2
+    assert tai.get_by_category("Decor")[0].condition == 2.0
+    assert tai.get_by_category("Clothing")[0].condition == 4.0
+    assert jesse.get_by_category("Clothing")[0].condition == 2.0
```

Comment:
I recommend restructuring these tests to make sure the correct elements are swapped.  It will make the asserts themselves a little simpler, but also check the specific action in the act step.  Also, removing `get_by_category` reduces the possibility that a bug in that function will falsely register a bug here.
```suggestion
    assert item_a in tai.inventory
    assert item_b in tai.inventory
    assert item_f in tai.inventory
    assert item_d in jesse.inventory
    assert item_e in jesse.inventory
    assert item_c in jesse.inventory
```